db/data/controllers/article.controller.js

```diff
@@ -0,0 +1,18 @@
+const allArticlesById = require('../models/articles.models')
+
+const getArticlesById = (request, response) => {
+    const article_id = parseInt(request.params.article_id)
+    console.log(request.params)
+         allArticlesById(article_id).then((article) => {
+            if(!article) {
```

Comment:
This should be handled in the model, this also isn't how to reject a promise. Refer back to the error handling notes on how to write a custom error.